Bottom: f7098734b90e3bb9a92221c046446891d12d7264
Top:    a4acfa9cd78aabcf2e9936c67eda92310132be0c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-08-07 23:10:40 -0700

Refresh of time-ex

---

diff --git a/Makefile.am b/Makefile.am
index 08582d3..4e3dee7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -29,7 +29,7 @@ src_librspreload_la_LIBADD = $(top_builddir)/src/librdmacm.la
 bin_PROGRAMS = examples/ucmatose examples/rping examples/udaddy examples/mckey \
 	       examples/rdma_client examples/rdma_server examples/rdma_xclient \
 	       examples/rdma_xserver examples/rstream examples/rcopy \
-	       examples/riostream examples/udpong
+	       examples/riostream examples/udpong examples/cmtime
 examples_ucmatose_SOURCES = examples/cmatose.c examples/common.c
 examples_ucmatose_LDADD = $(top_builddir)/src/librdmacm.la
 examples_rping_SOURCES = examples/rping.c
